funding/commitment_type_negotiation_test.go

```diff
@@ -39,6 +40,26 @@ func TestCommitmentTypeNegotiation(t *testing.T) {
 			expectsRes: lnwallet.CommitmentTypeAnchorsZeroFeeHtlcTx,
 			expectsErr: nil,
 		},
+		{
+			name: "local funder wants explicit, remote doesn't " +
+				"support so fall back",
+			localFunder: true,
+			channelFeatures: lnwire.NewRawFeatureVector(
+				lnwire.StaticRemoteKeyRequired,
+				lnwire.AnchorsZeroFeeHtlcTxRequired,
+			),
+			localFeatures: lnwire.NewRawFeatureVector(
+				lnwire.StaticRemoteKeyOptional,
+				lnwire.AnchorsZeroFeeHtlcTxOptional,
+				lnwire.ExplicitChannelTypeOptional,
+			),
+			remoteFeatures: lnwire.NewRawFeatureVector(
+				lnwire.StaticRemoteKeyOptional,
+				lnwire.AnchorsZeroFeeHtlcTxOptional,
+			),
+			expectsRes: lnwallet.CommitmentTypeAnchorsZeroFeeHtlcTx,
```

Comment:
nit: can be nil?

Reply:
Yep, fixed, and also re-introduced the old error type here since it's more apt imo.